### ISSUE TITLE
useOperators incompatible operators

### ISSUE THREAD

**Issue by vikiival** [kofrasa/mingo#169]
```ts
import { useOperators, OperatorType } from 'mingo/core'
import { $match, $group } from 'mingo/operators/pipeline'


useOperators(OperatorType.PIPELINE, { $match, $group })
```
I'm getting following error
```
33:47 Type '(collection: Iterator, expr: Record<string, unknown>, options?: Options | undefined) => Iterator' is not assignable to type 'Callback<unknown>'.
  Types of parameters 'collection' and 'args' are incompatible.
    Type 'unknown' is not assignable to type 'Iterator'.
    31 |
    32 | // tslint:disable-next-line
  > 33 | useOperators(OperatorType.PIPELINE, { $match, $group })
```

```
   "mingo": "^4.1.2",
```

**Comment by vikiival** [comment]
possible hack 
```ts
type OperatorMap = Record<string, any> ;
useOperators(OperatorType.PIPELINE, { $match, $group, $project } as OperatorMap)
```

**Comment by kofrasa** [comment]
Fixed in [4.1.3](https://github.com/kofrasa/mingo/releases/tag/4.1.3)
